assets/javascripts/modules/autoComplete.js

```diff
@@ -140,14 +140,16 @@ var updateInput = function (title, value) {
 };
 
 /**
- * Allow auto complete to control and pass over the .value of your suggestion to another optional input
+ * Allow auto complete to control and pass over the .value of your suggestion to one or more optional inputs
  * specified on creation of the autoComplete
+ * if the input is a select, pass the value, otherwise if there is a suggestionDisplayFormat, use that
  * @param value
  */
 var updateTargetInput = function (value) {
-  if ($targetInput) {
-    $targetInput.val(value);
-  }
+  var formattedString = (typeof suggestionDisplayFormat === 'function') ? suggestionDisplayFormat('',value) : value;
+  $.each($targetInputs, function (index, targetEl) {
+    $(targetEl).val($(targetEl).prop('tagName') === "SELECT" ? value : formattedString);
```

Comment:
`$(targetEl)` in some use cases will be wrapping a jQuery object here. `getTargetInput($elem)` and `$targetInputElem` provide jQuery elements to the `$targetElements` variable.